components/Login/Login.js

```diff
@@ -105,10 +40,11 @@ export default function Login({
       <StyledButton
         $position={position}
         onClick={() => {
-          if (position === "header") {
-            signIn();
-          } else if (position === "menu") {
-            setIsSigning("in");
+          signIn();
+          if (position === "menu") {
+            changeSubmenuMode("default");
+            changeFlashcardSelection("all");
+            handleToggleMenu();
           }
```

Comment:
same is true here, just use onClick:
```suggestion
onClick()
```